f2/src/forms/defaultValues.js

```diff
@@ -3,7 +3,7 @@ const formDefaults = {
   prodVersion: '',
   appVersion: '',
   consent: { consentOptions: [] },
-  anonymous: { anonymousOptions: ['anonymousPage.no'] },
```

Comment:
I think you changed default value  to "no" for load testing

Reply:
That is a good question 😅  - I just checked and the load testing is still fine.

Reply:
good to go.

Reply:
Excellent.

Reply:
Good to go